api-reference/docstubs/api/user-list-calendarview.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "List calendarView"
+description: "Get the events from the calendarView navigation property."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List calendarView
+
+Namespace: microsoft.graph
+
+Get the events from the calendarView navigation property.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /me/calendarView
```

Comment:
"HTTP request" section: 
This API takes on takes on mandatory query parameters for the start and end times like this:
?startDateTime={start_datetime}&endDateTime={end_datetime}
They don't seem to be in the metadata currently; calendarView is just defined as a navigation property.
Will we be able to automate similar kinds of API, or will this incur a manual modification after the topic is generated? 


"Query parameters" section:
Then per our [guidance](https://msgo.azurewebsites.net/add/document/guidelines/parameters.html) to document such parameters and, pointing customers to the topic for OData system query options so as not to have another section with a similar title "Optional query parameters", all these go under a separate H2 section titled "Query parameters".

Reply:
Need to investigate